### ISSUE TITLE
[master] ./configure --disable-x11 fails to compile with X11/Xlib.h: No such file or directory

### ISSUE THREAD

**Issue by hartwork** [FreeSpacenav/spacenavd#22]
```
In file included from src/dev.c:28:
src/proto_x11.h:23:10: fatal error: X11/Xlib.h: No such file or directory
   23 | #include <X11/Xlib.h>
      |          ^~~~~~~~~~~~
```
Probably the include to `proto_x11.h` in `src/dev.c` should be guarded by the X11 support macro.

Full log at https://708646.bugs.gentoo.org/attachment.cgi?id=612568

**Comment by jtsiomb** [comment]
fixed, commit: 0e7ee93

**Comment by hartwork** [comment]
Thank you!
